client/my-sites/plugins/plugins-discovery-page/index.jsx

```diff
@@ -146,38 +171,17 @@ const PluginsDiscoveryPage = ( props ) => {
 		category: 'featured',
 	} );
 
-	const requiredPlansPurchasedPlugins = useSelector( ( state ) =>
-		getPlansForFeature( state, props.selectedSite?.ID, WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS )
-	);
-	const requiredPlansAllPlugins = useSelector( ( state ) =>
-		getPlansForFeature( state, props.selectedSite?.ID, FEATURE_INSTALL_PLUGINS )
-	);
-
-	const hasInstallPurchasedPlugins = useSelector( ( state ) =>
-		siteHasFeature( state, props.selectedSite?.ID, WPCOM_FEATURES_INSTALL_PURCHASED_PLUGINS )
-	);
-
-	// Whether to show an upgrade banner specific to premium plugins.
-	const lowerPlanAvailable =
-		! hasInstallPurchasedPlugins &&
-		requiredPlansPurchasedPlugins[ 0 ] !== requiredPlansAllPlugins[ 0 ];
-
 	return (
 		<>
 			{ ! props.jetpackNonAtomic && (
 				<>
 					<div className="plugins-discovery-page__upgrade-banner">
```

Comment:
We can remove this div, dead code, though I wonder if it being dead is what caused this: https://github.com/Automattic/wp-calypso/issues/66484

Reply:
✅  - Didn't seem to fix the issue in #66484 though. :/